modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceEntryLocalServiceImpl.java

```diff
@@ -1480,6 +1486,29 @@ private GroupByStep _getGroupByStep(
 		);
 	}
 
+	private void _validatePrice(
+			BigDecimal discountLevel1, BigDecimal discountLevel2,
+			BigDecimal discountLevel3, BigDecimal discountLevel4,
+			BigDecimal price)
```

Comment:
Try to sort the variable order like CommercePriceEntry service.xml.

```suggestion
	private void _validatePrice(
			BigDecimal price, BigDecimal discountLevel1,
			BigDecimal discountLevel2, BigDecimal discountLevel3,
			BigDecimal discountLevel4)
```